vcf-dumper/vcf-dumper-lib/src/main/java/embl/ebi/variation/eva/vcfdump/VariantExporterController.java

```diff
@@ -54,44 +59,65 @@
 public class VariantExporterController {
 
     private static final Logger logger = LoggerFactory.getLogger(VariantExporterController.class);
-    private static final int WINDOW_SIZE = 20000;
+    private static final int WINDOW_SIZE = 10000;
```

Comment:
I need to measure exactly. In my laptop, Tomcat consumed too much memory dumping a 1000G query, so I had to low the window size. Maybe in a proper server this is not an issue, but I have to run some proper tests to know how much memory do a "big" dump (many variants condensed in one window and many samples) will consume.